VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts

```diff
@@ -31,9 +31,10 @@ export class PrivateConsultationRoomControlsComponent extends HearingControlsBas
         protected deviceTypeService: DeviceTypeService,
         protected logger: Logger,
         protected participantService: ParticipantService,
-        protected translateService: TranslateService
+        protected translateService: TranslateService,
+        protected userMediaService: UserMediaService
     ) {
-        super(videoCallService, eventService, deviceTypeService, logger, participantService, translateService);
+        super(videoCallService, eventService, deviceTypeService, logger, participantService, translateService, userMediaService);
```

Comment:
why are we injecting userMediaService here? doesnt look to be used in this component?

Reply:
It derives from hearing control base... :/ it could do with refactoring...

Reply:
ohh, I missed the "super" constructor, my bad.